lib/cfndsl/Types.rb

```diff
@@ -45,12 +39,15 @@ class AWSType < JSONable
     
     # Go through and declare all of the types first
     aws_types["Types"].each_key do |typename|
-      if( ! AWSTypes.const_defined? typename ) then
-        klass = AWSTypes.const_set( typename, Class.new(AWSType ) )
-        classes[typename] = klass
-      else
-        classes[typename] = AWSTypes.const_get(typename)
-      end 
+      next if %w(String JSON Integer).include?(typename)
```

Comment:
I had a reason at the time but can't remember. I wasn't getting any duplicates though.